pkg/util/strategicpatch/patch.go

```diff
@@ -66,8 +69,8 @@ type errConflict struct {
 	message string
 }
 
-func newErrConflict(patch, current []byte) errConflict {
-	s := fmt.Sprintf("patch:\n%s\nconflicts with current:\n%s\n", patch, current)
+func newErrConflict(patch, current string) errConflict {
+	s := fmt.Sprintf("patch:\n%s\nconflicts with automated changes:\n%s\n", patch, current)
```

Comment:
> Why is it valid to assume that this was changed by an automated process? It could have been any client that didn't know to update the magic annotation, right?

In fact, even kube clients of skewed versions will probably get this wrong because of version and codec changes, right?